node-commons/src/main/java/org/apache/iotdb/commons/schema/tree/AbstractTreeVisitor.java

```diff
@@ -834,4 +901,6 @@ protected IFAState tryGetNextState(N node, IFAState sourceState, IFATransition t
       return null;
     }
   }
+
+  protected abstract boolean isTargetNodeType(N node);
```

Comment:
```suggestion
  /**
   * May node can be accepted if it reaches final state. Its implementation should not depend on the
   * context.
   *
   * @param node node to be checked
   * @return false is if node must not be accepted. Otherwise, return true.
   */
  protected abstract boolean mayTargetNodeType(N node);
```

Reply:
Fixed.